{
  "generated_at": "2025-10-20T18:19:27.589206+08:00",
  "total_titles_processed": 1438,
  "failed_sources": [],
  "trends": [
    {
      "keyword_group": "chatgpt openai",
      "match_count": 5,
      "titles": [
        {
          "title": "OpenAI：所有受影响的服务现已完全恢复。",
          "url": "https://flash.jin10.com/detail/20251020181236971800",
          "source": "金十数据",
          "ranks": [
            3
          ],
          "is_new": true,
          "appearance_count": 1,
          "time_info": "18时19分"
        },
        {
          "title": "OpenAI：我们已发现用户在使用受影响的服务时遇到登录问题。",
          "url": "https://wallstreetcn.com/livenews/2990557",
          "source": "华尔街见闻-快讯",
          "ranks": [
            14,
            26
          ],
          "is_new": false,
          "appearance_count": 2,
          "time_info": "[17时19分 ~ 18时19分]"
        },
        {
          "title": "OpenAI：我们正面临“单点登录”（SSO）问题。",
          "url": "https://flash.jin10.com/detail/20251020164600924800",
          "source": "金十数据",
          "ranks": [
            18
          ],
          "is_new": false,
          "appearance_count": 1,
          "time_info": "17时19分"
        },
        {
          "title": "OpenAI：我们已发现用户在使用受影响的服务时遇到登录问题。",
          "url": "https://flash.jin10.com/detail/20251020164457909800",
          "source": "金十数据",
          "ranks": [
            19
          ],
          "is_new": false,
          "appearance_count": 1,
          "time_info": "17时19分"
        },
        {
          "title": "OpenAI：正在观察受影响服务的恢复情况，我们正在密切关注。",
          "url": "https://flash.jin10.com/detail/20251020174853570800",
          "source": "金十数据",
          "ranks": [
            28
          ],
          "is_new": true,
          "appearance_count": 1,
          "time_info": "18时19分"
        }
      ]
    }
  ],
  "report_image_url": "https://yyq83.github.io/TrendRadar/img/news.jpg"
}